documentation added for pipelines (#2793)
weaveworks/weave-gitops: website/docs/enterprise/pipelines/intro.mdx

---
title: Introduction
sidebar_position: 0
hide_title: true
---

import TierLabel from "../../_components/TierLabel";

<h1>
  {frontMatter.title} <TierLabel tiers="Enterprise" />
</h1>

## Pipelines

As [wikipedia defines](https://en.wikipedia.org/wiki/Continuous_delivery), Continuous delivery is

>a software engineering approach in which teams produce software in short cycles,
>ensuring that the software can be reliably released at any time and, when releasing the software, without doing so manually.
>It aims at building, testing, and releasing software with greater speed and frequency.

>Continuous delivery is enabled through the deployment pipeline.

> >The purpose of the deployment pipeline has three components: visibility, feedback, and continually deploy.[12]

Reviewer: ```suggestion
>The purpose of the deployment pipeline has three components: visibility, feedback, and continually deploy.
```

> Weave Gitops Enterprise Pipelines allows you to define your deployment pipelines to enable continuous delivery for

Reviewer: ```suggestion
Weave GitOps Enterprise Pipelines allows you to define your deployment pipelines to enable continuous delivery for
```

your gitops applications.

> As part of Weave Gitops Enterprise, you could use

Reviewer: ```suggestion
As part of Weave GitOps Enterprise, you can
```

- [Define a delivery pipeline](../getting-started/#define-a-pipeline) that you want you application to follow
- [View a pipeline](../getting-started/#view-pipeline-list) and check the status of your deployments.

> Now that you know what delivery pipelines could do for you, just [get started](../getting-started)

Reviewer: ```suggestion
Now that you know what delivery pipelines can do for you, follow the [guide to get started](../getting-started).
```
